contributors/devel/sig-instrumentation/migration-to-structured-logging.md

```diff
@@ -7,6 +7,61 @@ should be applicable at later stages or for other projects using `klog` logging
 
 [Structured Logging KEP]: https://github.com/kubernetes/enhancements/tree/master/keps/sig-instrumentation/1602-structured-logging
 
+## How to contribute
+
+### About the migration
+
+Migration proceeds in two different ways. First is organized milestone based large scale migration where we are trying to achieve a particular target for a Kubernetes release (for example migrate whole Kubelet code in 1.21). Second is not organized migration done by individual contributors.
+
+Organized migration is done in two state cycle aligned with Kubernetes releases. In stage one we pick a particular migration milestone and create issue that splits the work into smaller chunks (for example migrating just single file) that can be easily picked up by contributors. This way we ensure that multiple people can independently work on common goal and avoid conflicts. After migration is done we can utilize tooling we developed to mark particular directory as migrated to avoid regressions. When we finish particular milestone we move the second stage where we take a break from migration, to analyse results and improve on the process. Adding structured information to logs is very costly and time consuming process, we don't want to make a mistake that would require us to repeat the migration. That's why we reserve time to collect feedback, analyse impact of changes we have done on log volume, performance and PR review process.  
```

Comment:
Reworded as,

Organized migration is carried out in a two state cycle aligning with the cadence of Kubernetes releases. In the first stage, we pick a particular migration milestone & create an issue to split the work into smaller chunks (for example, migrating just a single file). This ensures that the work can be easily picked up by multiple individual contributors and also avoids conflicts. After the migration activity is complete, we mark the directory as migrated to avoid regressions with the help of tooling that we have developed. In the second milestone, we take a break from migration to analyze results & improve our existing processes. Adding structured information to logs is a very costly & time-consuming affair. Setting aside time in the second stage to collect feedback, analyze the impact of changes on the log volume & performance, and better our PR review process helps us avoid mistakes and duplicate efforts.